qemu/tests/cfg/multi_disk_random_hotplug.cfg

```diff
@@ -31,6 +31,9 @@
                 stg_params = "fmt:virtio,virtio_scsi,usb2"
             usbs += " ehci"
             usb_type_ehci = usb-ehci
+            Host_RHEL.m6:
+                usbs= "ehci"
+                usb_type_ehci = ich9-usb-ehci1
```

Comment:
@suqinhuang It uses ich9-usb-uhci1 after the PR change:

/usr/libexec/qemu-kvm \
    -S  \
......   
    -device ich9-usb-ehci1,id=ehci,addr=1d.7,multifunction=on,bus=pci.0 \
    -device ich9-usb-uhci1,id=ehci.0,multifunction=on,masterbus=ehci.0,addr=1d.0,firstport=0,bus=pci.0 \
    -device ich9-usb-uhci2,id=ehci.1,multifunction=on,masterbus=ehci.0,addr=1d.2,firstport=2,bus=pci.0 \
    -device ich9-usb-uhci3,id=ehci.2,multifunction=on,masterbus=ehci.0,addr=1d.4,firstport=4,bus=pci.0 \
......
    -device usb-tablet,id=usb-tablet1,bus=ehci.0,port=1  \
    -vnc :0  \
    -rtc base=utc,clock=host,driftfix=slew  \
    -boot order=cdn,once=c,menu=off,strict=off \
    -enable-kvm
......
{'execute': 'device_add', 'arguments': OrderedDict([('driver', 'usb-storage'), ('id', 'stg10'), ('drive', 'drive_stg10'), ('bus', 'ehci.0'), ('port', 6)]), 'id': 'BmUs1I2w'}